_layouts/team-page.html

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Should this file exist? I think you may have picked the wrong outcome of the merge conflict resolver.